### ISSUE TITLE
Fix traceback when trying to download a recording without subscription

### ISSUE THREAD

**Issue by tchellomello** [tchellomello/python-ring-doorbell#3]
```bash
>>> myring.doorbell_recording_url(123456778)
Sorry.. Something went wrong...
Traceback (most recent call last):
   File "<stdin>", line 1, in <module>
  File "/home/frank/code/python-ring-doorbell/ring_doorbell/__init__.py", line 235, in doorbell_recording_url
    if req.status_code == 200:
AttributeError: 'NoneType' object has no attribute 'status_code'
```

@tchellomello If someone tries to download/get the URL of a video while they have no active subscription it throws traceback:

Thanks to @syphernl for reporting the issue.

**Comment by tchellomello** [comment]
@syphernl could you share the output for the command below on your environment?

```python
from ring_doorbell import Ring
myring = Ring('user@me.com', 'secret')
myring.features
```

Mine looks like this:

 ```python
In [3]: myring.features
Out[3]: 
{'chime_dnd_enabled': False,
 'chime_pro_enabled': True,
 'delete_all_enabled': True,
 'delete_all_settings_enabled': False,
 'device_health_alerts_enabled': True,
 'live_view_settings_enabled': True,
 'lpd_enabled': True,
 'lpd_motion_announcement_enabled': False,
 'multiple_calls_enabled': True,
 'multiple_delete_enabled': True,
 'nw_enabled': True,
 'nw_user_activated': False,
 'owner_proactive_snoozing_enabled': True,
 'power_cable_enabled': False,
 'proactive_snoozing_enabled': False,
 'reactive_snoozing_enabled': False,
 'remote_logging_format_storing': False,
 'remote_logging_level': 1,
 'ringplus_enabled': True,
 'starred_events_enabled': False,
 'stickupcam_setup_enabled': True,
 'subscriptions_enabled': True,
 'ujet_enabled': False,
 'vod_enabled': False}
```

**Comment by tchellomello** [comment]
Should be fixed on 0.0.2
